application/uitests/org.openjdk.jmc.flightrecorder.uitest/src/test/java/org/openjdk/jmc/flightrecorder/uitest/ControlRecordingsTest.java

```diff
@@ -247,6 +248,7 @@ public void stopTimeFixedRecording() {
 	/**
 	 * Verifies that recording events can be added/removed on the fly
 	 */
+	@Ignore("JDK-8286740 did not fix the issue on Windows")
```

Comment:
JDK-8286740 was the backport, now that we're on JDK 21 the correct issue would be: https://bugs.openjdk.org/browse/JDK-8328159